src/openzaak/components/zaken/openapi.yaml

```diff
@@ -3041,6 +3111,45 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Zaak'
+                  inclusions:
```

Comment:
@sergei-maertens I noticed that inclusions were also added to the `ZaakZoek` response schema (despite it not actually working for `ZaakZoek` currently). I assume that `include` should also be allowed for `ZaakZoek` operations?